### ISSUE TITLE
Key functionality broken with new patch

### ISSUE THREAD

**Issue by ireliamonster** [Verubato/framesort#16]
When I play shuffles now, I am no longer reliably at the top of the raidframes

I would assume this is the case for everybody, but let me know if I should help reproduce the issue

**Comment by ireliamonster** [comment]
Traditional seems to work reliably, Taintless does not

**Comment by Verubato** [comment]
Oh snap, looking into it.

**Comment by Verubato** [comment]
Do you by chance have 2 sets of visible party frames? e.g. both the Blizzard and GladiusEx party frames shown?

**Comment by Verubato** [comment]
Just released a new version 5.5.2 which might have fixed this.

**Comment by ireliamonster** [comment]
> Do you by chance have 2 sets of visible party frames? e.g. both the Blizzard and GladiusEx party frames shown? Do you use GladiusEx at all?

Yes, that was probably it. Works with new update. Good catch, amazing quick update :)
